static/js/zcommand.js

```diff
@@ -191,6 +241,42 @@ export function process(message_content) {
         return true;
     }
 
+    if (content.indexOf("/digress") === 0) {
+        if (compose_state.get_message_type() !== "stream") {
+            // digress can be used only in streams
+            return true;
+        }
+
+        let param = content.slice(8);
+        param = param.trim();
+
+        // A valid param will now be of the form #**stream name>topic name**
+        const new_stream_name_start = param.indexOf("#**") + 3;
+        const new_topic_start = param.indexOf(">") + 1;
+        const new_topic_end = param.lastIndexOf("**");
```

Comment:
Would it be better to use `marked(param)` (which will return something like `<p><a class="stream-topic" data-stream-id="14" href="/#narrow/stream/14-test/topic/topic">#test &gt; topic</a></p>`) and then use jQuery to fetch the stream-id and topic?